src/components/ScrollTrack/ScrollTrack.js

```diff
@@ -17,9 +17,21 @@ class ScrollTrack extends Component {
   static ScrollTrackPropTypes = ScrollTrackPropTypes
 
   static propTypes = {
+    /** Prop for passing in custom button content for back button */
+    backButtonContent: React.PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.Symbol,
```

Comment:
Nitpick - does this work (PropTypes.Symbol)? https://reactjs.org/docs/typechecking-with-proptypes.html suggest it's `PropTypes.symbol`, though I've usually used `PropTypes.node` in the past for react elements. I'm actually not sure what the correct way is.